src/features/DisplayControls/DrugChart/utils/DrugChartUtils.js

```diff
@@ -88,41 +88,57 @@ export const TransformDrugChartData = (drugChartData) => {
           administeredTime = moment(administeredDateTimeObject).format(
             hourFormatString
           );
+          const performer = providers.find(provider => provider.function === performerFunction).provider;
```

Comment:
find method return undefined in case the element is not found, can you put the corresponding checks?
let performer = providers.find(provider => provider.function === performerFunction);
performer = performer ? performer.provider : {};

Reply:
refactored as advised